Continuous Delivery and Operation Excellence/Continuous Delivery.md

```diff
@@ -0,0 +1,54 @@
+# Content gathered for Continuous Delivery
+
+## Delivery formats

```

Comment:
FYI, needs blank line after ## to be standards compliant MD. :grin:

Reply:
Fixed